README.md

```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
 yarn
 ```
 
+### Environment variables
+
+To get started local development, create a file in the root of the repo named
+`.env` with the following content.
```

Comment:
Add the sentence:

"You can also set the variables in your shell environment."